api/v1/report.proto

```diff
@@ -31,11 +31,14 @@ message ReportRequest {
   // within the current stream.
   map<string, string> facts = 2;
 
+  // The monitored resource type associated with this report.
+  string monitored_resource_descriptor_name = 3
+
   // Metric values being reported
-  repeated MetricValue metric_values = 3;
+  repeated MetricValue metric_values = 4;
```

Comment:
Does this imply that the proxy will populate metrics and log_entries and send to mixer?  I think a better design is: the proxy just send the primary data, such as size, duration, ip, and mixer will populate the metrics and log-entries based on its backend plugins.